### PR TITLE
HARVESTER: revert namespace to default manually

### PR DESCRIPTION
**Pull request by DaiYuzeng**
#### PR Checklist
- ~Is this a multi-tenancy feature/bug?~
    - ~[ ] Yes, the relevant RBAC changes are at:~
- ~Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?~
    - ~[ ] Yes, the relevant PR is at:~
- ~Are backend engineers aware of UI changes?~
    - ~[ ] Yes, the backend owner is:~

### Occurred changes and/or fixed issues

- https://github.com/harvester/harvester/issues/3517

### Areas or cases that should be tested

1. Go to create Secret/ Logging Cluster Flow/ Logging Cluster Output in `cattle-logging-system` namespace
2. Go to Volume create page
3. The preset namespace should not be `cattle-logging-system`

image

### CODE REVIEW

File: pkg/harvester/edit/harvesterhci.io.logging.clusteroutput.vue [hunk 1]
```diff
@@ -7,6 +8,12 @@ export default {
     if (this.isCreate) {
       this.value.metadata.namespace = 'cattle-logging-system';
     }
+
+    this.registerAfterHook(() => {
+      const ns = this.$store.getters['allNamespaces']?.[0] || '';
```
Comment (WuJun2016) [inline pkg/harvester/edit/harvesterhci.io.logging.clusteroutput.vue:13]: ditto

File: pkg/harvester/edit/harvesterhci.io.logging.clusterflow.vue [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+export default {
+  extends: Flow,
+  created() {
+    if (this.isCreate && this.value.type === LOGGING.CLUSTER_FLOW) {
+      this.value.metadata.namespace = 'cattle-logging-system';
+    }
+
+    this.registerBeforeHook(this.willSave, 'willSave');
+    this.registerAfterHook(() => {
+      const ns = this.$store.getters['allNamespaces']?.[0] || '';
```
Comment (WuJun2016) [inline pkg/harvester/edit/harvesterhci.io.logging.clusterflow.vue:15]: ditto

File: pkg/harvester/edit/harvesterhci.io.logging.clusterflow.vue [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
+  created() {
+    if (this.isCreate && this.value.type === LOGGING.CLUSTER_FLOW) {
+      this.value.metadata.namespace = 'cattle-logging-system';
+    }
+
+    this.registerBeforeHook(this.willSave, 'willSave');
+    this.registerAfterHook(() => {
+      const allNamespaces = this.$store.getters['allNamespaces'];
+      const defaultNamepsace = allNamespaces.find(N => N.id === 'default');
+      const ns = defaultNamepsace?.id || allNamespaces?.[0]?.id || '';
```
Comment (DaiYuzeng) [inline pkg/harvester/edit/harvesterhci.io.logging.clusterflow.vue:17]: Please take a re-review, thanks! @WuJun2016

**Review by WuJun2016 — APPROVED** [review]
LGTM， thanks

**Pull request merged**